### ISSUE TITLE
[60] As an Instructor, I would like to be notified of new learning materials added or updated in the collection so that I'm up-to-date on OER and OEP

### ISSUE THREAD

**Issue by bdolor** [BCcampus/open-sage#60]
**Conditions of Satisfaction**

New and updated icons on the textbook covers

1. New is only "new" for 60 days
2. Updated is valid after new has expired and the modified date is within 60days

"Find Open Textbooks" landing page
Pg. 4 in open.Bccampus.ca template markup

![image](https://user-images.githubusercontent.com/2048170/52677584-4d3fee00-2ee3-11e9-8a01-d46787c434cc.png)

**Comment by bdolor** [comment]
@ehbeattie @dshaykewich If the user story swapped the word `see` to `be notified of` then this user story would be broad enough to invite discussion around how best to achieve that...so adding labels would be one way to achieve that, but not the only way. 

Either way, what's missing from this is what `New` means, and what `Updated` means in terms of time. How long does something stay `New` for and how long does it stay `Updated` for. Other considerations are if a new book is posted, then updated shortly after, what is the desired behaviour?

**Comment by ehbeattie** [comment]
@bdolor I agree with the change in terminology from _'see'_ to _'be notified of'_.

**Comment by ehbeattie** [comment]
@bdolor - the desired behaviour should show the most recent changes to the collection, whether it's an addition, modification or update. The duration is negotiable.

**Comment by ehbeattie** [comment]
@josieg - can you comment on this one?

**Comment by josieg** [comment]
`New` means a book is added to the collection for the first time.

`Updated` means there has been a noteworthy* change made to the book. (with "noteworthy" meaning files have been replaced or a new resources has been added - books that have only been updated minorly, i.e. small error fix, changes to the book's decription in SOLR should not be marked as `updated`) **--- Question: Is it even possible to differentiate?**

How long does something stay `New` for and how long does it stay `Updated` for?
- What if rather than setting a time, we set a number? For example, the last three books added to the collection are marked as `New` and last two books updated are marked as `updated`

If a new book is posted, then updated shortly after, what is the desired behaviour?
- I think the `New` label should take priority over the `Updated` label
